### PR TITLE
Introducing Metrics in InnerSource: Merge PR497 and PR501 upstream (instead of new patterns)

### PR DESCRIPTION
**Pull request by fioddor**
[PR497](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/497#issuecomment-1336526141) duplicates this pattern.
[PR501](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/501) might extend this pattern instead of creating a new one.
This MR intends to capture the useful bits and merge them upstream so PR497 and/or PR501 can be withdrawn.

### CODE REVIEW

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 1]
```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
 * Some organizations in some countries may face extra complexity when introducing metrics as the countries may not allow tracking individuals.
 * There might be a learning curve in the discussion about metrics. And perhaps the tools do not support the InnerSource metrics we are looking for.
 
+In favour:
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
In favor:
```

Changing to AE spelling

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 2]
```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
...
+
+* Management needs to understand how InnerSource is impacting development. Metrics allow for accurate visualizations.
+* There are several industry standards for monitoring the software development process (DORA, CHAOSS, ...).
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
* There are several industry standards for monitoring the software development process (e.g. [DORA](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance), [CHAOSS](https://chaoss.community/)).
```

I don't want to assume that everybody knows these metrics, so I added to links to relevant resources.
Especially for DORA I am not sure if there is an authoritative source on this, but I figured Google SRE should be a good start.

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 3]
```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
...
 
+### Manage people's needs
+
 Bring developers, middle managers and C-level to have a discussion about metrics. And consider other roles out of the usual development process such as Human Resources, legal departments, product management, and others.
 
 Let developers and middle managers know that these metrics or KPIs are not focused on tracking their personal performance, but to compare if the initiative is currently working as expected.
 
 Consider a third party that is seen as a neutral player to produce such metrics.
 
+### Approach professionaly
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
### Approach professionally
```

Spelling

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 4]
```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
...
+
 Have specific training on the topic of metrics and good practices to use them. An example is to have a methodology to follow metrics such as the Goal-Question-Metric approach or the Objectives-KeyResults one. On the other hand, try to reflect the short-term and medium-term goals in the metrics to be used.
 
-Metrics when published or discussed should be done so in the aggregate without referring to specific people.
+Metrics when published or discussed should
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
When publishing or discussing metrics they should be
```

I find it easier to read in this form

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 5]
```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
...
+
+* be aggregated, without referring to specific people,
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
* aggregated - without referring to specific people
```

this change only makes sense if the previous line is also changed as suggested

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 6]
```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
...
+* watching out for outliers.
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md]:
```suggestion
* watching out for outliers
```

File: patterns/1-initial/introducing-metrics-in-innersource.md [hunk 7]
```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
...
 
 Produce a characterization of metrics as this might be helpful for others to understand and follow.
```
Comment (spier) [inline patterns/1-initial/introducing-metrics-in-innersource.md:62]: Do you have an example of what is meant by "characterization"?

**Review by spier — COMMENTED** [review]
@fioddor thanks for consolidating the different PRs related to this pattern. You approach sounds good to me.

I left some comments inline. Will merge the straightforward fixes myself.

Aside:
As you are adding a Known Instance to this pattern, we could upgrade this pattern to Structured soon. I suggest to do that via a separate PR though, once all the others are merged or closed. That allows us to do one more proper review and check the pattern against all requirements of the Structured level.

File: patterns/1-initial/introducing-metrics-in-innersource.md (hunk 7 above)
Comment (fioddor) [reply patterns/1-initial/introducing-metrics-in-innersource.md:62]: It's an old text. [It was there before the move in jul 2020](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/aecdd1821b0f091e1a88c8318df52c58fdfa789b/patterns/2-structured/introducing-metrics-in-innersource.md). I guess it refers to the next 2 bullet points.

Comment (spier) [reply patterns/1-initial/introducing-metrics-in-innersource.md:62]: Oh yeah, didn't see the bullet's right behind it. Thanks!

**Review by spier — APPROVED** [review]
LGTM

**Pull request merged**